### ISSUE TITLE
Not able to open

### ISSUE THREAD

**Issue by theapache64** [romainguy/kotlin-explorer#3]
![image](https://github.com/romainguy/kotlin-explorer/assets/9678279/4d9de308-b8f2-4894-aff3-033cd57693f0)

**Comment by theapache64** [comment]
@romainguy Yes. Is the `dmg` signed and notarized ? -> https://github.com/JetBrains/compose-multiplatform/blob/ef04feee54f1d757114f0811c59052447c584767/tutorials/Signing_and_notarization_on_macOS/README.md

**Comment by romainguy** [comment]
No it's not and I won't do this unfortunately.

**Comment by theapache64** [comment]
@romainguy Then I think there's no point sharing a `dmg` file 😬 . Can you share the `jar` file? That shouldn't have this issue.

**Comment by romainguy** [comment]
It can be opened, you first need to right click > Open then confirm again by clicking Open in the dialog. But like mentioned in the README file you need to first make sure you have set `ANDROID_HOME` and `KOTLIN_HOME`. I just double-checked on a different computer where I never compiled/ran the app and it works just fine with the latest macOS. Alternatively you can run `KOTLIN_HOME=/opt/homebrew/Cellar/kotlin/1.9.0/libexec open /Applications/Kotlin\ Explorer.app` from a terminal for instance.
